frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -85,15 +95,27 @@ export class ReduxTlmDispatcher extends React.Component {
     }
   }
 
+  handleWorkspaceCreated = data => {
+    const { props } = this
+    if (ACCESSIBLE_SPACE_TYPE_LIST.find(s => s.slug === data.fields.workspace.access_type) !== undefined) {
```

Comment:
alternatively, you can use `Array.some()` which returns a boolean instead of using `Array.find() !== undefined`